web_listview_range_select/__manifest__.py

```diff
@@ -1,22 +1,20 @@
-# © 2017 Onestein (<http://www.onestein.eu>)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
 {
     "name": "List Range Selection",
     "summary": """
         Enables selecting a range of records using the shift key
     """,
-    "version": "15.0.1.0.0",
+    "version": "16.0.1.0.0",
     "category": "Web",
-    "author": "Onestein, Odoo Community Association (OCA)",
+    "author": "Synodica Solutions Pvt. Ltd., Onestein, Odoo Community Association (OCA)",
```

Comment:
Put your company after the original author.

Reply:
Changed company position after original author.